7.middleware/README.md

```diff
@@ -37,12 +41,13 @@ app.get('/echo', (req, res) => {
     console.log("Echoing: " + text);
     res.send("Access token: " + req.headers["authorization"] + " Text: " + text)
 });
+```
 
 ```

```

Comment:
Please remove either of these two lines (L44 or L46)

Reply:
umh, I think it makes sense that we close this PR and I reopen a new PR with the ingress chart change. I tried to rebase after I saw that the repo was renamed and the structure changed. Above issue came through back-merging, because the fix I wanted to introduced was obviously in the repo structure change PR.